phoebe/frontend/bundle.py

```diff
@@ -3780,8 +3781,10 @@ def run_checks_compute(self, compute=None, solver=None, solution=None, figure=No
                                     True, 'run_compute')
 
             # NOTE: atms are not attached to datasets, but per-compute and per-component
-            for atmparam in self.filter(qualifier='atm', kind='phoebe', compute=computes, **_skip_filter_checks).to_list() + self.filter(qualifier='ld_coeffs_source').to_list():
-
+            # NOTE: atmparam includes a '_default' compute pset, which depends on the
+            # ck2004 atmospheres; the checks should not include it. This is achieved
+            # by filtering check_visible=True in the line below:
+            for atmparam in self.filter(qualifier='atm', kind='phoebe', compute=computes, check_visible=True, check_default=False).to_list() + self.filter(qualifier='ld_coeffs_source').to_list():
```

Comment:
woops, I actually think I got these Trues and Falses backwards.  We want to skip the visible checks but include the default checks so that `_default` are excluded from the returned parameters entirely (didn't notice it until I saw your comment explanation! 😬 )

Reply:
ok, so do we want to do it this way then? As this way buries the logic deeper, it might be difficult to parse. Even now I'm not sure: if check_visible=True, I'd interpret that to only take visible psets, which would exclude _default. Now you're saying that check_visible should be False? And then that check_default should be True? That seems counter-intuitive to me.

Reply:
Yes, it will be (marginally) cheaper, so I think since this runs often it is worth the explanatory comment and keeping the code as efficient as possible.

Reply:
the problem is that even the explanatory comment doesn't help me understand the logic. ;) But okay, I'll recommit with the fix and move on.

Reply:
`check_visible` checks whether a parameter is visible, and if it is not, excludes it from the results.  `check_default` similarly checks whether a parameter has any tag (compute, component, dataset, etc) as `_default`, and if so, excludes it from the results.  In this case you do NOT want parameters that are tagged default, so we need to use `check_default=True` (or skip the parameters with a continue).  We are not worried about parameters that are not visible/relevant, so we can save some time by passing `check_visible=False`.